src/benchmarks/micro/libraries/System.Runtime/Perf.Array.cs

```diff
@@ -273,43 +285,27 @@ public void ArrayResize()
                 Array.Resize<byte>(ref _byteArrays[i], NewSize);
         }
 
-        private readonly int[] _reversibleArray = Enumerable.Range(0, 256).ToArray();
+        [GlobalSetup(Target = nameof(Reverse))]
+        public void SetupReverse() => _reversibleArray = Enumerable.Range(0, 256).ToArray();
 
         [Benchmark]
         public void Reverse() => Array.Reverse(_reversibleArray);
 
-        [GlobalSetup(Target = nameof(ClearUnaligned))]
-        public void SetupClearUnaligned()
-        {
-            while (true)
-            {
-                var buffer = new byte[8192];
-                GCHandle handle = GCHandle.Alloc(buffer, GCHandleType.Pinned);
-                if (((long)handle.AddrOfPinnedObject()) % 32 != 0)
-                {
-                    _clearableArrayHandle = handle;
-                    _clearableArray = buffer;
-                    return;
-                }
-                handle.Free();
-            }
-        }
- 
-        [GlobalCleanup(Target = nameof(ClearUnaligned))]
-        public void CleanupClearUnaligned() => _clearableArrayHandle.Free();
+        [GlobalSetup(Target = nameof(Clear))]
+        public void SetupClear() => _clearableArray = new byte[8192];
  
-        private GCHandle _clearableArrayHandle;
-        private byte[] _clearableArray;
-
         [Benchmark]
-        public void ClearUnaligned() => Array.Clear(_clearableArray, 0, _clearableArray.Length);
+        public void Clear() => Array.Clear(_clearableArray, 0, _clearableArray.Length);
```

Comment:
Are we getting rid of the ClearUnaligned test here?

Reply:
Yes, as we should rather force BDN to perform the memory alignment randomization instead of having a setup that tries to allocate an unaligned array ([source code](https://github.com/dotnet/performance/blob/4e14ecc54759224121b807c70e46fb1d9ee0e7d6/src/benchmarks/micro/libraries/System.Runtime/Perf.Array.cs#L281-L296)) (so we have always unaligned memory compared to having aligned and unaligned)